### PR TITLE
853503, Ивойлов, лабораторная 2.2 (UI design).

### PR DESCRIPTION
**Pull request by am-moving-lightspeed**
GitHub pages links:
* [main page](https://gotspeed.github.io/Caravan/);
* ['about' page](https://gotspeed.github.io/Caravan/about);
* [stage of forming a deck](https://gotspeed.github.io/Caravan/choosing-cards);
* [in-game process](https://gotspeed.github.io/Caravan/game);
* ['sign-in' page](https://gotspeed.github.io/Caravan/sign-in);
* ['sign-up' page](https://gotspeed.github.io/Caravan/sign-up);

### CODE REVIEW

**Comment by am-moving-lightspeed** [comment]
The markdown and styles might look a bit messy, but I'm going to tidy up later, when will be doing JS part (if you let me). Also I'm gonna change some elements and styles/animations, because, I guess, that it's easier to come up with what I'll need when I'll be doing JS part of particular pages. For now I'll leave the project as it is.
Really hope you won't kill me for what I've done in this lab. :c

File: Caravan/about.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <article class="about-article">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+            incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+            exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+            dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+            Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+            mollit anim id est laborum.
+        </article>
+    </main>
+    <button class="back-button"></button>
```
Comment (gmltA) [inline Caravan/about.html:25]: U sure it should not be a link?

**Review by gmltA — APPROVED** [review]
Quite unusual. That's a <kbd>9</kbd>. Looking forward to see the result.

File: Caravan/about.html (hunk 1 above)
Comment (am-moving-lightspeed) [reply Caravan/about.html:25]: No, this is this button. It does not have a :hover effect, but has an :active. Anyway, It's to be replaced.

image

Appreciated for `9`. Pretty unexpected...

**Pull request merged**